apps/pkcs12.c

```diff
@@ -878,15 +879,45 @@ int cert_load(BIO *in, STACK_OF(X509) *sk)
     return ret;
 }
 
+/* Generalised x509 attribute value print */
+
+void print_attribute(BIO *out, const ASN1_TYPE *av)
+{
+    char *value;
+    switch (av->type) {
```

Comment:
NIT: blankline

Reply:
@mattcaswell - do you want this change in 1.1.1?

Reply:
I am thinking this is fairly dubiously a "bug", and is more a missing feature. Which means it wouldn't qualify for backport.

Reply:
I'm of the opposite opinion.  If there are several attributes, what would be the reason to only display the first?  I suspect sloppy programming driven by available PKCS#12 files available at the time.  Hence, bug.

Reply:
Ok. I'm not entirely convinced but I'm also not going to object - although I would prefer if someone else did the approvals.

Reply:
Not a problem, I can do that.

Reply:
> Ok. I'm not entirely convinced but I'm also not going to object - although I would prefer if someone else did the approvals.

@mattcaswell does this mean you want to dismiss your approval, or shall it remain together with @levitte's and mine?